Source/ACE/Network/NetworkSession.cs

```diff
@@ -309,6 +309,9 @@ private void SendBundle(NetworkBundle bundle)
                             currentMessageFragment = new MessageFragment(bundle.Messages.Dequeue());
                         }
 
+                        //possibly an issue with messages being null
+                        //uncomment to test: if (currentMessageFragment.message == null) break;
```

Comment:
I am noticing that this crashes my server when the message is null, because you can't have a length of a null member?